Assets/MRTK/SDK/StandardAssets/Sprites/DefaultIconsSpriteSheet.psd.meta

```diff
@@ -1,21 +1,45 @@
 fileFormatVersion: 2
 guid: 20f25674b9512834f88938d45ed495d7
 TextureImporter:
-  fileIDToRecycleName:
-    21300000: AppBarAdjust
-    21300002: AppBarClose
-    21300004: AppBarHide
-    21300006: AppBarShow
-    21300008: ButtonFollowMe
-    21300010: ButtonHandJoint
-    21300012: ButtonHandMesh
-    21300014: ButtonHandRay
-    21300016: ButtonHome
-    21300018: ButtonPin
-    21300020: ButtonProfiler
-    21300022: AppBarDone
+  internalIDToNameTable:
```

Comment:
Looks like this file was reserialized in Unity 2019. We should keep it in the 2018 format for now.

Reply:
I thought I managed to remove it qq...
reverting